### PR TITLE
samosa edit

### PR DESCRIPTION
**Pull request by RazVan2208**
samosa samosa edit #1

### CODE REVIEW

File: samosa.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-chicken 
\ No newline at end of file
+chicken 
+is
+yummy
```
Comment (tahmid0517) [inline samosa.txt]: Put the above 3 lines on the same line.

File: samosa.txt [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+i want a samosa now :(
```
Comment (tahmid0517) [inline samosa.txt:14]: Add a :skull

File: samosa.txt (hunk 1 above)
Comment (tahmid0517) [reply samosa.txt]: Explain why chicken is yummy,

**Review by tahmid0517 — CHANGES_REQUESTED** [review]

File: samosa.txt (hunk 1 above)
Comment (RazVan2208) [reply samosa.txt]: done, enjoy the nutritional chicken

File: samosa.txt (hunk 2 above)
Comment (RazVan2208) [reply samosa.txt:14]: done enjoy skull

**Review by tahmid0517 — APPROVED** [review]
lgtm

**Pull request merged**